### DISCUSSION TITLE
Hold to show workspace overview

### DISCUSSION THREAD

**Discussion by kevjqs** [mzur/gnome-shell-wsmatrix#260] [Ideas]
Hi there,

I've been using this extension for a while and I love it.

Just one little question: 

Currently the default shortcut to show workspace overview popup is Super+W, but it doesn't get dismissed unless you press ESC. To make viewing workspace overview more fluid, and more efficient, is it possible to allow the workspace overview popup to show for as long as a key is **held** and when the key is released the popup is dismissed?

Thanks

**Comment by mzur** [comment]
You can get the behavior you describe by using the <kbd>Ctrl</kbd>+<kbd>Alt</kbd>+<kbd>Arrow</kbd> popup while setting the popup timeout to 0 in the extension settings. The popup will be visible for as long as the keys are pressed.

**Reply by kevjqs** [reply]
Thanks! It works great :D
